data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+	on complete
+		payment 60000
+	on visit
+		dialog `You need to bring Daemos, his cargo, and the Gonza Book Cruise here to complete this mission.`
+	on decline
+		fail "Alexandria Library Card: Rescue: Escort"
+	on fail
+		fail "Alexandria Library Card: Rescue: Escort"
+	npc accompany save
+		personality escort derelict timid target
+		government "Librarian"
+		ship "Star Queen" "Gonza Book Cruise"
+		system Mora
+	npc
+		government "Pirate"
+		personality staying plunders disables
+		system Mora
+		fleet "Small Southern Pirates" 3
+		fleet "Large Northern Pirates" 2
+
+
+
+
+mission "Alexandria Library Card: Rescue: Escort"
+	name `Escort Fleet: Deep Security`
+	description `A Deep Security rescue fleet will escort you to Mora so you can repair the Gonza Book Cruise before <date>.`
+	source Valhalla
+	destination Ingot
+	deadline 9
+	to offer
+		has "Alexandria Library Card: Return: done"
+		not "Alexandria Library Card: Rescue: declined"
+		not "Alexandria Library Card: Rescue: failed"
+	on offer
+		conversation
+			`Deep Security meets you and Daemos. "A fleet is prepared, but only your ship can board the Gonza Book Cruise. We must bring it back to <destination> before <date>."`
+				accept
+	to complete
+		has "Alexandria Library Card: Rescue: done"
+	npc
+		government "Deep Security"
+		personality heroic escort disables
+		fleet
+			names "deep"
+			variant
+				"Bactrian (Hired Gun)"
+				"Dagger" 3
+				"Corvette"
+				"Corvette (Speedy)"
+				"Headhunter" 3
+				"Aerie" 1
+				"Dagger" 2
+
+	
+
+mission "Alexandria Library Card: Completion"
+	name "Alexandria Library Card"
+	description "Take Dameos Titan to <destination> to archive the Chal Jub Paqmey."
+	blocked `Daemos complains, "I can't sleep on a bookshelf. I need a bunk and space for 1 ton of cargo."`
+	source Ingot
+	destination Alexandria
+	passengers 1
+	landing
+	clearance
+	cargo "old books and a thousand pencils" 1
+	to offer
+		has "Alexandria Library Card: Rescue: done"
+	on offer
+		conversation
+			`When you arrive at Ingot, you see some Seekers and Deep Security agents conversing with Daemos.`
+			`	Daemos greets you with 60,000 credits and some kind words. "Just get these five books and I back to Alexandria and our Seek will be complete. You'll be a Journeyman Seeker with all the associated rights and privileges... and 100,000 more credits."`
+			choice
+				`	"Great! Let's leave."`
+					accept
+				`	"I'm done with your quest, old man. I don't want rights or privileges, and I certainly don't want 100,000 credits."`
+					decline
+	on visit
+		dialog `You've forgotten your senile friend or his pencils. Perhaps they're in one of your escorts?`
+	on complete
+		set "library card"
+		payment 100000
+		set "license: Alexandria Seeker"
+		conversation
+			`When you land at the station, a gathering of librarians meet you at your ship. Daemos ceremoniously pins a gold icon of a book onto you and hands you a key card allowing access to the station and its library.`
```

Comment:
Slight repetition of "you".
```suggestion
			`When you land at the station, a gathering of librarians meet you at your ship. Daemos ceremoniously pins a gold icon of a book to your shirt and hands you a key card allowing access to the station and its library.`
```

Reply:
I'd rather leave the ambiguity of where the pin was placed. Also, in this context, "onto you" can imply it is going onto something you are wearing.